packages/nextjs/src/server/wrapApiHandlerWithSentry.ts

```diff
@@ -11,44 +11,26 @@ import {
 } from '@sentry/utils';
 import * as domain from 'domain';
 
-import type { AugmentedNextApiRequest, AugmentedNextApiResponse, NextApiHandler, WrappedNextApiHandler } from './types';
-import { formatAsCode, nextLogger } from './utils/nextLogger';
+import type { AugmentedNextApiRequest, AugmentedNextApiResponse, NextApiHandler } from './types';
 import { platformSupportsStreaming } from './utils/platformSupportsStreaming';
 import { autoEndTransactionOnResponseEnd, finishTransaction, flushQueue } from './utils/responseEnd';
 
 /**
  * Wrap the given API route handler for tracing and error capturing. Thin wrapper around `withSentry`, which only
  * applies it if it hasn't already been applied.
  *
- * @param maybeWrappedHandler The handler exported from the user's API page route file, which may or may not already be
+ * @param apiHandler The handler exported from the user's API page route file, which may or may not already be
  * wrapped with `withSentry`
  * @param parameterizedRoute The page's route, passed in via the proxy loader
  * @returns The wrapped handler
  */
-export function wrapApiHandlerWithSentry(
-  maybeWrappedHandler: NextApiHandler | WrappedNextApiHandler,
-  parameterizedRoute: string,
-): WrappedNextApiHandler {
-  // Log a warning if the user is still manually wrapping their route in `withSentry`. Doesn't work in cases where
-  // there's been an intermediate wrapper (like `withSentryAPI(someOtherWrapper(withSentry(handler)))`) but should catch
-  // most cases. Only runs once per route. (Note: Such double-wrapping isn't harmful, but we'll eventually deprecate and remove `withSentry`, so
-  // best to get people to stop using it.)
```

Comment:
I decided to remove this because it would have been a pain to get working with proxies and it doesn't hurt anyone not to have this log message.